mod/01_rating/set/abstract/all_values.rb

```diff
@@ -0,0 +1,182 @@
+include_set Type::SearchType
```

Comment:
it's all "all_metric_values" now (for companies and metrics in rating and caching mod) except this abstract set used by the "all_metric_values" sets